### ISSUE TITLE
P384 support

### ISSUE THREAD

**Issue by raphaelrobert** [facebook/voprf#81]
Since P384 is mandated as the only valid ciphersuite by [Privacy Pass](https://www.ietf.org/archive/id/draft-ietf-privacypass-protocol-06.html#name-client-to-issuer-request), it would be nice to have support for it.

I suppose it is blocked by https://github.com/RustCrypto/traits/pull/1024.

**Comment by daxpedda** [comment]
I can attempt to push this forward in a different way so we wouldn't have to wait for the next version of `elliptic-curve` if this is blocking you.

**Comment by daxpedda** [comment]
With #101 merged the [`p384`](https://crates.io/crates/p384) crate is now supported.
#84 will add tests to `voprf`.

Currently neither `voprf` or `p384` have a crates.io release for P-384 support though.
